docs/docs/extension.md

```diff
@@ -295,6 +295,106 @@ The `auth` object allows 3 different auth mechanism:
 - `oauth`
 - `custom`
 - `cloudentity`
+- `jwt`
+
+#### JWT
+
+For further details, please see the [JWT](./guides/authentication/jwt.md) guide.
+
+##### Definition
+
+**TODO(MBana):** Complete section.
```

Comment:
Can we finish the docs before merging?

Reply:
@aabedraba and @jasmingacic chare to chime in here?

@kylehodgetts, I don't mind doing the documentations now but it's going to change a lot, I suspect. Can we leave it to a future revision as I suspect the JWT support/configuration is going to change a lot as we iterate.

Reply:
Im just thinking if we release whatever's on the main branch, then this will also get released in its current state. We need to make sure the docs are in place before we do another release